src/main/java/org/springframework/integration/aws/config/xml/S3InboundChannelAdapterParser.java

```diff
@@ -50,5 +50,4 @@ protected String getMessageSourceClassname() {
 	protected Class<? extends InboundFileSynchronizer> getInboundFileSynchronizerClass() {
 		return S3InboundFileSynchronizer.class;
 	}
-
```

Comment:
No. Blank like before the last curly brace in the class: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style.
But, eh, that's minor. Will fix on merge :smile: